src/main/resources/templates/db_scripts/schema.json

```diff
@@ -289,6 +289,48 @@
           "targetTableAlias": "fiscalYear"
         }
       ]
+    },
+    {
+      "tableName": "groupFundFY",
+      "fromModuleVersion": 4.0,
+      "foreignKeys": [
+        {
+          "fieldName": "groupId",
+          "targetTable": "groups",
+          "tableAlias": "groupFundFY",
+          "targetTableAlias": "group"
+        },
+        {
+          "fieldName": "fundId",
+          "targetTable": "fund",
+          "tableAlias": "groupFundFY",
+          "targetTableAlias": "fund"
+        },
+        {
+          "fieldName": "fiscalYearId",
+          "targetTable": "fiscal_year",
+          "tableAlias": "groupFundFY",
+          "targetTableAlias": "fiscalYear"
+        },
+        {
+          "targetPath": ["fundId", "ledgerId"],
+          "targetTable": "ledger",
+          "targetTableAlias": "ledger",
+          "tableAlias": "groupFundFY"
+        },
+        {
+          "targetPath": ["fundId", "fundTypeId"],
+          "targetTable": "fund_type",
+          "targetTableAlias": "fundType",
```

Comment:
`"targetTableAlias": "fundType"` does not exists? It might work because it is optional. However, I cannot find where it says it is optional - https://github.com/folio-org/raml-module-builder#cql2pgjson-foreign-key-cross-table-index-queries. Should we add an alias first before referring it?

Reply:
Could you please provide a bit more details on what is wrong in this definition? The configuration is done based on [Foreign key tableAlias and targetTableAlias](https://github.com/folio-org/raml-module-builder/tree/v27.0.0#foreign-key-tablealias-and-targettablealias) document and verified by unit test using following query:
> query=group.code==HIST and fundType.name==Monographs and ledger.name==One-time

Reply:
I thought targetTableAlias for example here `fundType` should exist and only then it can be assigned to targetTableAlias. But, that is not the case, targetTableAlias is created at runtime while executing CQL.